grc/main.py

```diff
@@ -28,8 +28,27 @@
     'critical': logging.CRITICAL,
 }
 
+#DEBUG
+import sys
+import inspect
+
+def audit(event, args):
+    global EVALUATED, EVALUATED_PATH
+
+    if event == 'exec':
+        stack = inspect.stack()
+        path = stack[1].filename
+        #print(*stack, sep="\n")
+
+        #if "/grc/core" in path:
+        if "/grc/" in path:
+            print("IN GRC")
+            print(*stack, sep="\n")
+
```

Comment:
Maybe remove this, now that is in the tests?

Reply:
yeah, somehow missed that it was still there